### PR TITLE
Improve metrics info display

### LINKED ISSUE #683: [FEATURE] Improve metrics info display
Today we have metrics info kind of ugly 😛:
Image

It should look like this:
Image

Resource:
https://drive.google.com/file/d/1YG68PXIfzKExCQn5d-Ho0raoayy9C1-z/view?usp=drive_link

### PR DESCRIPTION
**Pull request by ncontinanza**
Closes #683 

This PR implements a refactor of the metrics that we show at the bottom of the screen. You can find a reference picture in the issue.

To test it, launch a new game and verify the changes in the new bar.

### CODE REVIEW

File: client/Assets/Scenes/BackendPlayground.unity
Comment (BertovDev) [inline client/Assets/Scenes/BackendPlayground.unity:1]: If you only made changes in the CameraUi prefab remember to discard changes made in the scene.

If you made the changes in the prfab inside the scnee you have to go to the section override in the prefab inside the scene. Then Apply All and discard the Scene changes.

File: client/Assets/Scripts/VersionsHashes.cs [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
 		}
 
-        string server_hash = "Server hash \t" + SocketConnectionManager.Instance.serverHash;
-        string client_hash = "Client hash \t" + GitInfo.GetGitHash();
-		GetComponent<Text>().text = server_hash + "\n" + client_hash;
-	}
-
-	public void Update()
-	{
+        string hash = serverHash ? "Server# " + SocketConnectionManager.Instance.serverHash : "Client# "+ GitInfo.GetGitHash();
```
Comment (BertovDev) [inline client/Assets/Scripts/VersionsHashes.cs:16]: This should not be like this : `"Server #"`? same for the client

**Review by BertovDev — CHANGES_REQUESTED** [review]
Looks nice! I left you some comments. I will wait for your response so for now i will request changes 😔

**Review by BertovDev — DISMISSED** [review]
Awesome!!!

**Review by BertovDev — APPROVED** [review]

**Review by AngieDutra — APPROVED** [review]
This looks great! ✨

**Pull request merged**